src/renderer/components/+namespaces/add-dialog/dialog.tsx

```diff
@@ -29,7 +30,7 @@ export interface AddNamespaceDialogProps extends DialogProps {
 
 interface Dependencies {
   namespaceStore: NamespaceStore;
-  model: AddNamespaceDialogModel;
+  state: IObservableValue<boolean>;
```

Comment:
This is movement to wrong direction, as ideally the abstraction of a UI-component should not know of technical stuff like "state". A ViewModel is preferable, because it can provide steady abstractions, with apparent intent, all customised for consumption of a single view.

Reply:
Whereas I disagree since the models are monolithic and are too broad in general. The dialog doesn't need to know how to `open` itself and those who want to open the dialog don't need to know how to `close`.

Reply:
The view-model is a long topic for someplace else. To the future! Resolved.